app/home.vmd

```diff
@@ -16,7 +16,7 @@ path: home
 Vale's goal is to show the world that speed and safety can be easy. Vale is:
 
  * *Fast:* Vale uses an entirely new approach to memory management: [generational references](/blog/generational-references), which have zero aliasing costs and no garbage collection pauses.
- * *Fearless:* It is the [safest native language](/blog/fearless): zero `unsafe`, region isolation, extern boundaries, and dependency extern whitelisting.
+ * *Fearless:* It is the [safest native (AOT compiled) language](/blog/fearless): zero `unsafe`, region isolation, extern boundaries, and dependency extern whitelisting.
```

Comment:
Good call on including this! Can we move it up a line, to say "Vale is an AOT compiled language that uses an entirely new ..."

Putting it here seems like it could be narrowing this sentence's claim.

Reply:
yes that is better